website/docs/getting-started/setup/upgrade-to-business-edition/README.md

```diff
@@ -1,29 +1,42 @@
+---
+description: Follow the steps to upgrade the Appsmith installation to a Business Edition.
+---
 # Upgrade to Business Edition
 
-Follow the below guides to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)** based on the type of your **Appsmith Installation:**
+Follow the steps below to **upgrade** your **Community Edition (CE)** to a **Business Edition (BE)**. Choose the type of  **Appsmith Installation** to get started.
 
-:::note
-You'll need your  [**Business Edition License Key**](https://www.appsmith.com/pricing)  to complete the upgrade.
+:::info License Key
+You would need a **Business Edition License Key** to complete the upgrade. Learn more on [**Business Edition & pricing**](https://www.appsmith.com/pricing) options.
 :::
 
-
-* [Docker](docker.md)
-
-
-:::note
-For **non-docker Appsmith installations**, please get in touch with us via [discord](https://discord.com/invite/rBTTVJp) or [email](mailto:support@appsmith.com). We will help you upgrade your Appsmith instance.
-:::
-
-## License Renewal
-
-The Appsmith server shuts down if your Business license key is expired. You will see an invalid license key message like below in the logs:
-
-```
+<div class="containerBorder">
+    <div class="containerGrid">
+        <div class="columnGrid column-one" align="center">
+            <div class="containerCol">
+                <a href="docker/">
+                <img class="containerImage" src="/img/docker-logo.png" alt="Docker-logo"/>
+                </a> 
+            </div> 
+            <b><a href="docker/">Docker</a></b>
+        </div>
+        <div class="columnGrid column-two" align="center">
+       </div>
+        <div class="columnGrid column-three" align="center">
+        </div>
+    </div>
+</div>
+
+## License renewal
+
+The Appsmith server shuts down once the license key expires. You would see `Invalid license key` message in the logs:
+
+```bash
 Exiting application. Invalid license key.
 ```
 
-:::info
-Please get in touch with our support team either on [Discord](https://discord.com/invite/rBTTVJp) or by [email](mailto:support@appsmith.com) to renew your license.
-:::
+Raise a request on [Discord](https://discord.com/invite/rBTTVJp), or [send an email to support](mailto:support@appsmith.com) to raise a renewal request. Once you have a renewed license key, [replace](docker#step2-add-or-replace-license-key) it to have Appsmith up and running.
```

Comment:
I think we should stick to either method for renewal requests. Support Email would be better than Discord as the threads can be tracked easily. On Discord, it can get all mixed up. So suggestion to start the line with 

> Send an email to [support](mailto:support@appsmith.com) to raise a renewal request. Once you have a new license key, replace the key in the configuration file to start the Appsmith server.

I think we shouldn't cross-reference 'replace' to the Docker instructions because we are going to have other guides up here as well

@sharat87 what do you think?

Reply:
Yes, true. But until the time we don't have other docs, we can keep it for quick access.